data/json/items/comestibles.json

```diff
@@ -1684,9 +1684,115 @@
     "weight": 225,
     "color": "white",
     "symbol": "%",
-    "description": "A bone from some creature or other.  Could be used to make some stuff, like needles.",
+    "description": "A bone from some creature or other.  Could be used to make some stuff, like needles, or cracked open for marrow.",
     "price": 0,
     "material": "bone",
+    "volume": 1,
+    "use_action": {
+      "target": "bone_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "menu_text": "Extract marrow",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "bone_marrow",
+    "name": "bone marrow",
+    "weight": 110,
+    "color": "white",
+    "spoils_in": 120,
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": 2,
+    "nutrition": 15,
+    "description": "A soft, spongy substance found in bones. Chock-full of good stuff.",
+    "price": 0,
+    "charges": 2,
+    "material": "bone",
+    "volume": 1,
+    "vitamins": [ [ "calcium", 5 ], [ "iron", 10 ] ]
+  },
+  {
+    "type": "GENERIC",
+    "id": "bone_tainted",
+    "category": "other",
+    "name": "tainted bone",
+    "weight": 212,
+    "color": "light_gray",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": -1,
+    "nutrition": 2,
+    "description": "A rotten and brittle bone from some unnatural creature or other.",
+    "price": 0,
+    "//": "Not for use in edible recipes, and should require ~200% as much as normal for applicable inedible recipes except for charcoal.",
+    "material": "bone",
+    "volume": 1,
+    "fun": -10,
+    "use_action": {
+      "target": "tainted_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "menu_text": "Extract marrow",
+      "type": "transform"
+    }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "tainted_marrow",
+    "name": "tainted marrow",
+    "weight": 100,
+    "color": "white",
+    "spoils_in": 60,
+    "use_action": "POISON",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": -1,
+    "nutrition": 5,
+    "description": "A wet, green chunk of spongy goop extracted from a rotten bone. You could eat it, if you were desperate.",
+    "price": 0,
+    "charges": 2,
+    "material": "bone",
+    "volume": 1
+  },
+  {
+    "type": "GENERIC",
+    "id": "bone_human",
+    "category": "spare_parts",
+    "name": "human bone",
+    "weight": 225,
+    "color": "white",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "healthy": -1,
+    "nutrition": 4,
+    "description": "A bone from a human being.  Could be cracked into marrow or used to make some stuff, if you're feeling sufficiently ghoulish.",
+    "price": 0,
+    "//": "Not for use in normal food recipes.",
+    "material": [ "bone", "hflesh" ],
+    "volume": 1,
+    "use_action": {
+      "target": "human_marrow",
+      "msg": "You snap the bone into two pieces and extract the marrow.",
+      "menu_text": "Extract marrow",
```

Comment:
Missing `"target_charges": 2,` otherwise this will create [bone marrow item with 0 charges](https://cdn.discordapp.com/attachments/365907426270117888/474686597086838806/0bonemarrowcharges.png) and [prompt error when attempting to eat it](https://cdn.discordapp.com/attachments/365907426270117888/474686599494500374/bonemarrowerror.png).